usagov/vote-gov-nvrf-app
# VOTE-2910 Replace Spanish PDF and revert workaround

https://cm-jira.usa.gov/browse/VOTE-2910

Description:
1. Replace the Spanish PDF with the latest from eac.gov
2. Revert the duplicate GeneratePDF function workaround in favor of a shared function.

Test steps:
1. Visit the Spanish NVRF (http://localhost:5173/es/)
2. Complete all fields (use the update reg path) and confirm all fields correctly map  to fields in the generated Spanish PDF.
3. Repeat for English.
